galasa-parent/buildSrc/src/main/groovy/galasa.api.server.gradle

```diff
@@ -9,9 +9,9 @@ dependencies {
     implementation 'org.osgi:org.osgi.service.component.annotations:1.3.0'
     implementation 'javax.validation:validation-api:2.0.1.Final'
     implementation 'commons-logging:commons-logging:1.2'
-    implementation 'com.google.code.gson:gson:2.8.9'
+    implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
Has gson been changed everywhere ? Think it's used elsewhere too ?

Reply:
Also bumped in dev.galasa.framework - had a check in other repos and it looks like it'll also need bumping in managers and extensions